app/views/quotes/_quote.html.erb

```diff
@@ -1,7 +1,9 @@
+<%= quote.quoted_person %> said, <q><%= quote.body %></q>, <%= quote.location %>.
```

Comment:
This kind of thing could live in a presenter. something like <%= quote.formatted_quote %> 

Here is an article about presenters and why they are cool: http://nithinbekal.com/posts/rails-presenters/